camayoc/api.py

```diff
@@ -121,30 +126,88 @@ def __init__(self, response_handler=None, url=None):
         else:
             self.response_handler = response_handler
 
+        if authenticate:
+            self.login()
+            self._authenticated = True
```

Comment:
This can be dropped, the authenticated state is already being set by line 146

Reply:
I guess I just can't assert on this in the test, because when I mock the function, `_authenticated` does not get set